cli/block/service.py

```diff
@@ -0,0 +1,44 @@
+"""
+Provide implementation of the block.
+"""
+import asyncio
+
+from accessify import implements
+from aiohttp_json_rpc import RpcGenericServerDefinedError
+
+from cli.block.interfaces import BlockInterface
+
+loop = asyncio.get_event_loop()
+
+
+@implements(BlockInterface)
+class Block:
+    """
+    Implements block.
+    """
+
+    def __init__(self, service):
+        """
+        Constructor.
+
+        Arguments:
+            service: object to interact with Remme core API.
+        """
+        self.service = service
+
+    def get(self, identifier):
+        """
+        Get a block by its identifier.
+        """
+        try:
+            block_information = loop.run_until_complete(
```

Comment:
Leave just the `block`, because we have a `get_block_info` method that suits this value better.

Reply:
@anastasiia-bilova, done.